server/index-node/src/resolver.rs

```diff
@@ -204,12 +204,27 @@ where
                 .map_err(SubgraphManifestResolveError::ResolveError)?;
             match kind {
                 BlockchainKind::Ethereum => {
-                    UnvalidatedSubgraphManifest::<graph_chain_ethereum::Chain>::resolve(
+                    // FIXME (NEAR): Commented out for now otherwise it creates a compilation error because both
+                    //               types are incompatible in the match arms.
+                    // UnvalidatedSubgraphManifest::<graph_chain_ethereum::Chain>::resolve(
+                    //     deployment_hash,
+                    //     raw,
+                    //     self.link_resolver.clone(),
+                    //     &self.logger,
+                    //     MAX_SPEC_VERSION.clone(),
+                    // )
+                    // .await?
+                    unimplemented!()
```

Comment:
This should not panic, since it was working before!

Reply:
Probably a left over before your refactoring, will adjust.

Reply:
I think it was really having a compilation issue that I was not able to solve, might need your help on this one.